README.md

```diff
@@ -409,6 +410,55 @@ Returns the element that has the matching value.
 const lastNameInput = getByValue('Norris')
 ```
 
+### `getByDisplayValue`
+
+```typescript
+getByDisplayValue(
+  container: HTMLElement,
+  value: TextMatch,
+  options?: {
+    exact?: boolean = true,
+    collapseWhitespace?: boolean = false,
+    trim?: boolean = true,
+  }): HTMLElement
+```
+
+Returns the element that has the matching display value.
+
+#### `input`
+
+```javascript
+// <input type="text" id="lastName" />
+// document.getElementById('lastName').value = 'Norris'
+
+const lastNameInput = getByDisplayValue('Norris')
+```
+
+#### `textarea`
+
+```javascript
+// <textarea id="messageTextArea"></textarea>
+// document.getElementById('messageTextArea').value = 'Hello World'
+
+const messageTextArea = getByDisplayValue('Hello World')
+```
+
+#### `select`
+
+```javascript
+// <select id="state-select" data-testid="state">
+//   <option value="">State</option>
+//   <option value="AL">Alabama</option>
+//   <option selected value="AK" >Alaska</option>
+//   <option value="AZ">Arizona</option>
+// </select>
+
+const selectElement = getByDisplayName('Alaska')
+console.log(selectElement.id)
```

Comment:
Remove the `console.log` here (it's not necessary and it's nice to be consistent with the others). Then fix it to be:

> const selectElement = getByDisplayName(document.body, 'Alaska')